_src/static/sass/components/_buttons.scss

```diff
@@ -0,0 +1,371 @@
+    padding: .8rem 1.2rem;
+}
+
+
+// ----------------------------
+// #LARGE
+// ----------------------------
+.btn-large {
+    font-size: font-size(large);
+    padding: 1rem 1.4rem;
+}
+
+
+// ----------------------------
+// #SMALL
+// ----------------------------
+.btn-small {
+    font-size: font-size(small);
+    padding: .6rem 1rem;
+}
+
+
+// ----------------------------
+// #DEFAULT
+// ----------------------------
+.btn-default {
+    background: palette(grayscale, white);
+    border-color: palette(primary, mid-dark);
+    color: palette(primary, mid-dark);
+
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, mid-dark);
+        color: palette(grayscale, white);
+    }
+
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, dark);
+        border-color: palette(primary, dark);
+    }
+
+    &:disabled,
+    &.is-disabled {
+        background: palette(grayscale, white);
+        border-color: palette(grayscale, x-light);
+        color: palette(grayscale, base);
+    }
+}
+
+
+// ----------------------------
+// #DEFAULT-ALT
+// ----------------------------
+.btn-default-alt {
+    background: $transparent;
+    border-color: palette(grayscale-cool, base);
+    color: palette(grayscale-cool, base);
+
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, mid-dark);
+        border-color: palette(primary, mid-dark);
+        color: palette(grayscale, white);
+    }
+
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, dark);
+        border-color: palette(primary, dark);
+    }
+
+    &:disabled,
+    &.is-disabled {
+        background: $transparent;
+        border-color: palette(grayscale-cool, x-light);
+        color: palette(grayscale-cool, mid-light);
+    }
+}
+
+
+// ----------------------------
+// #PRIMARY
+// ----------------------------
+.btn-primary {
+    background: palette(primary, mid-dark);
+    border-color: palette(primary, mid-dark);
+    color: palette(grayscale, white);
+
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, base);
+        border-color: palette(primary, base);
+    }
+
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, mid-dark);
+        border-color: palette(primary, mid-dark);
+    }
+
+    &:disabled,
+    &.is-disabled {
+        background: palette(grayscale, x-light);
+        border-color: palette(grayscale, x-light);
+        color: palette(grayscale, base);
+    }
+}
+
+
+// ----------------------------
+// #SECONDARY
+// ----------------------------
+.btn-secondary {
+    background: $transparent;
+    border-color: $transparent;
+    color: palette(primary, base);
+
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        border-color: palette(grayscale-cool, trans);
+    }
+
+    &:active,
```

Comment:
NIT: add stateful Sass comments - https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#sass-comments-for-development